app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditActivity.kt

```diff
@@ -8,6 +8,8 @@ import javax.inject.Inject
 
 const val PROFILE_EDIT_PROFILE_ID_EXTRA_KEY = "ProfileEditActivity.profile_edit_profile_id"
 const val IS_MULTIPANE_EXTRA_KEY = "ProfileEditActivity.is_multipane"
+const val IS_PROFILE_DELETION_DIALOG_VISIBLE_KEY =
+  "ProfileEditActivity.is_profile_deletion_dialog_visible"
```

Comment:
@rt4914 Just want to confirm if it is the best way of using const keys or we should file an issue to replace this.
I believe that they should be declared like this -
```kotlin
const val KEY_NAME = "KEY_NAME"
```

Reply:
Actually this is the correct way.